students/elmar_m/lesson09/test_mailroom.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+
+'''
+file: test_mailroom.py
+elmar_m / 22e88@mailbox.org
+Lesson09: unittests for mailroom.py class
+'''
+
+
+import unittest 
+from classes_mailroom import Mailroom, Donor
+
+class Mailroom_Tests(unittest.TestCase):
+    
+    def test_Donor(self):   
+        # obj = mailroom.Donor('John', 'Doe')
+        obj = Donor('John', 'Doe')
+        self.assertIs(obj.firstname, 'John')
+        self.assertIs(obj.lastname, 'Doe')
+        self.assertEqual(obj.uid, 'John_Doe')
+
+    
+    def test_Collection(self):
+        nice_person = Donor('John', 'Doe')
+
+        db = Mailroom()
+        # db.add_donor(nice_person.uid)
```

Comment:
since you are using DBs to store data you would normally have `setUp` and `tearDown` methods that put your tests in the same clean state every time a test is run.